README.md

```diff
@@ -197,7 +197,7 @@ The default main layout of the [application template](https://github.com/yiisoft
 And the view template of the main page (`site/index`) will be as follows:
 
 ```twig
-{{ this.setTitle(applicationParameters.getName()) }}
+{% set this = this.setTitle(applicationParameters.getName()) %}
```

Comment:
```suggestion
{% do this.setTitle(applicationParameters.getName()) %}
```

Reply:
Agreed: I'd assumed WebView was immutable, but as it's not `do` is right

Reply:
`WebView` has state (see [here](https://github.com/yiisoft/view#state-of-view-and-webview-services)). `set*` methods is mutable, `with*` methods is immutable.